index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Conference Page</title>
+    <link rel="stylesheet" href="styles/index-style-max-width-767px.css">
+    <link rel="stylesheet" media="screen and (min-width:768px)" href="styles/index-style-min-width-768px.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/1d827eb1c8.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-top">
+            <ul class="social-networks">
+                <li><a href="https://www.facebook.com/"><i class="fab fa-facebook-f" title="Facebook"></i></a></li>
+                <li><a href="https://twitter.com/"><i class="fab fa-twitter" title="Twitter"></i></a></li>
+            </ul>
+            <div class="languages">English</div>
+            <div id="my-page">My Page</div>
+            <div class="logout">Logout</div>
+        </div>
+        <div class="navbar-bottom">
+            <a href="./index.html"> <img id="navbar-logo" src="./images/conference-logo.PNG" alt="Global Summit Logo"> </a>
+            <ul class="navbar-sections">
+                <li><a href="pages/about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><img class="close" src="./images/times.svg" alt="close menu icon"></li>
+            </ul>
+            <button id="at-campaign" type="button">CC Campaign</button>
+        </div>
+    </nav>
+
+    <section id="introduction">
+        <img class="toggleNav open" src="./images/ic_menu.svg" alt="Hamburguer icon">
+        <div id="title">
+            <h2>"Hello! Sharing World"</h2>
+            <h1>Agricultural Technology Global Summit 2022</h1>
+        </div>
+        <p>An exciting experience for both agriculture and tech
+            enthusiasts is taking place in March, in Costa Rica.</p>
+        <div class="when-where">
+            <h2 id="date">2022.03.15(TUE) ~ 16(WED)</h2>
+            <h2 id="location">@ Centro de Convenciones de Costa Rica</h2>
+        </div>
+    </section>
+
+    <section id="program">
+        <h2>Main Program</h2>
+        <ul id="program-list">
+            <li id="lecture">
+                <i class="fas fa-chalkboard-teacher"></i>
+                <h3>Lecture</h3>
+                <p>Listen to industry professionals from around the globe
+                    speak about the newest tendencies.</p>
+            </li>
```

Comment:
- [ ] [REQUIRED] I highly recommend that you stick to the colors used in the original design because considering the project requirements we need to make sure that your design looks almost identical to the original design.

👉🏻 Basic color swatch: #272A31, #EC5242, #D3D3D3